FreeBSD make error - Fails to build due to tcl.h header location - Linux Syndrome

```
cc -fPIC -I/usr/local/include/python3.11 -I/usr/local/include/python3.11  -Wsign-compare -Wunreachable-code -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing  -DNDEBUG -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing  -pipe -Wall -I. -I../../.. -I../../..  -I../../../src/mod -I/usr/include -DHAVE_CONFIG_H -I/usr/local/include/tcl8.6  -DMAKING_MODS -c .././python.mod/python.c && mv -f python.o ../
In file included from .././python.mod/python.c:46:
.././python.mod/pycmds.c:25:10: fatal error: 'tcl8.6/tcl.h' file not found
   25 | #include <tcl8.6/tcl.h>
      |          ^~~~~~~~~~~~~~
1 error generated.
*** Error code 1
```

See https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=283584

Part of #176.

Related to python.mod.

Maybe related to #1740.

This bug is interesting for i spent a lot of time making / keeping eggdrop BSD friendly and feel real pain when our child is diagnosed with "Linux Syndrome".

--- a/src/mod/python.mod/configure.ac
+++ b/src/mod/python.mod/configure.ac
@@ -1,6 +1,6 @@
 dnl configure.ac: this file is processed by autoconf to produce ./configure.
 
-AC_PREREQ(2.69)
+AC_PREREQ(2.71)
 
 sinclude(../eggmod.m4)
 builtin(include,../../../m4/python.m4)

--- a/src/mod/python.mod/pycmds.c
+++ b/src/mod/python.mod/pycmds.c
@@ -20,10 +20,6 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 #define PY_SSIZE_T_CLEAN
-#include <Python.h>
-#include <datetime.h>
-#include <tcl.h>
-#include "src/mod/module.h"
 
 typedef struct {
   PyObject_HEAD

--- a/src/mod/python.mod/python.c
+++ b/src/mod/python.mod/python.c
@@ -36,7 +36,6 @@
 #include "src/mod/server.mod/server.h"
 #include "python.h"
 
-//static PyObject *pymodobj;
 static PyObject *pirp, *pglobals;
 
 #undef global

--- a/src/mod/python.mod/python.h
+++ b/src/mod/python.mod/python.h
@@ -1,4 +1,3 @@
 #define PY_SSIZE_T_CLEAN
 #include <Python.h>
-#define PY_OK 0
 #undef putserv

